stdlib/public/SwiftShims/SwiftStdint.h

```diff
@@ -21,6 +21,18 @@
 // Clang has been defining __INTxx_TYPE__ macros for a long time.
 // __UINTxx_TYPE__ are defined only since Clang 3.5.
 
+#if defined(_WIN32)
```

Comment:
Is `_WIN32` the right check here? I was expecting a check for clang versus MSVC.